@here/harp-mapview/lib/PickHandler.ts

```diff
@@ -165,6 +166,9 @@ export class PickHandler {
             }
 
             const featureData: TileFeatureData = intersect.object.userData.feature;
+            if (this.enablePickTechnique) {
```

Comment:
Will this work for SolidLines techniques? As I see TileGeometry creator does not add technique to `userData` in this case?

Reply:
Ok found it, it's OK.